DNN Platform/Library/Security/Roles/RoleController.cs

```diff
@@ -654,6 +654,22 @@ public static bool DeleteUserRole(UserInfo objUser, RoleInfo role, PortalSetting
             return canDelete;
         }
 
+        /// <summary>
+        /// Completely remove all a user's roles for a specific portal. This method is used when 
+        /// anonymizing a user
+        /// </summary>
+        /// <param name="user">User for which all roles must be deleted. The PortalId property
+        /// is used to determine for which portal roles must be removed.</param>
+        public static void DeleteUserRoles(UserInfo user)
```

Comment:
Same question on this one about possible internal/private

Reply:
Done